### ISSUE TITLE
Test runner infinite load after migration to V10

### ISSUE THREAD

**Issue by karenpetrie** [cypress-io/cypress#22312]
### Current behavior

When following the migration steps to upgrade to V10, I completed the migration steps and then get the following loading page which just spins infinitely. 

![image](https://user-images.githubusercontent.com/16258312/173623745-d7d3e591-8874-4857-8db1-7f13bbc51503.png)

### Desired behavior

The test runner should load and display the tests

### Test code to reproduce

Firstly I ran:
`npm install cypress@latest`

When the latest version was installed I then ran:
`npx cypress open`

### Cypress Version

V10.1.0

**Comment by ChiNguyen2022** [comment]
I also got the same problem when trying to complete the migration 
![image](https://user-images.githubusercontent.com/107401595/173765458-ad2b29f8-b5b6-4c7e-9026-0aeb7b5c28a2.png)

**Comment by bobgager** [comment]
I'm having the same issue.
Terminal in VSC lists out the following error when I launch Cypress:

@ cypress:open /Users/bob/Google Drive/Cobalt Cloud/Developement/VSCprojects/Cypress/cypress
> cypress open

objc[6088]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x7ffb4b6ce948) and /Users/bob/Library/Caches/Cypress/10.1.0/Cypress.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x1118f79c8). One of the two will be used. Which one is undefined.
/Users/bob/Library/Caches/Cypress/10.1.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_plugins.js:40
  invoke = (eventId, args = []) => {
         ^

SyntaxError: Unexpected token =
    at new Script (vm.js:80:7)
    at createScript (vm.js:274:10)
    at Object.runInThisContext (vm.js:326:10)
    at Module._compile (internal/modules/cjs/loader.js:664:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/bob/Library/Caches/Cypress/10.1.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/child/run_require_async_child.js:6:24)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)

**Comment by ChiNguyen2022** [comment]
Here is my output
PS D:\Working\MyProject> set DEBUG=cypress:*
PS D:\Working\MyProject> npx cypress open

C:\Users\a\AppData\Local\Cypress\Cache\10.1.0\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_plugins.js:40
  invoke = (eventId, args = []) => {
         ^

SyntaxError: Unexpected token =
    at new Script (vm.js:79:7)
    at createScript (vm.js:251:10)
    at Object.runInThisContext (vm.js:303:10)
    at Module._compile (internal/modules/cjs/loader.js:657:28)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\a\AppData\Local\Cypress\Cache\10.1.0\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_require_async_child.js:6:24)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
[13976:0620/104812.703:ERROR:gpu_init.cc(446)] Passthrough is not supported, GL is disabled, ANGLE is

**Comment by aryzing** [comment]
I too am facing an infinite loading screen on a recent install of Cypress, may be worth reopening this issue?

My setup:

* Node v18.7.0
* Cypress,

```
Cypress package version: 10.3.1
Cypress binary version: 10.3.1
Electron version: 18.3.0
Bundled Node version: 16.13.2
```

* Config file, `cypress.config.ts`, (this was autogenerated by Cypress when first `open`ing it)

```ts
import { defineConfig } from "cypress";

export default defineConfig({
  e2e: {
    setupNodeEvents(on, config) {
      // implement node event listeners here
    },
  },
});
```

This is the output of running `DEBUG=cypress:* yarn cypress open`,

<details>
<summary>Show logs</summary>

```
  cypress:cli:cli cli starts with arguments ["/home/aryzing/.nvm/versions/node/v18.7.0/bin/node","/home/aryzing/workspace/stacks-challenge/node_modules/.bin/cypress","open"] +0ms
  cypress:cli NODE_OPTIONS is not set +0ms
  cypress:cli:cli program parsing arguments +1ms
  cypress:cli:cli opening Cypress +0ms
  cypress:cli parsed cli options {} +58ms
  cypress:cli verifying Cypress app +0ms
  cypress:cli checking environment variables +0ms
  cypress:cli checking if executable exists /home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress +2ms
  cypress:cli Binary is executable? : true +1ms
  cypress:cli binaryDir is  /home/aryzing/.cache/Cypress/10.3.1/Cypress +1ms
  cypress:cli Reading binary package.json from: /home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/package.json +0ms
  cypress:cli Found binary version 10.3.1 installed in: /home/aryzing/.cache/Cypress/10.3.1/Cypress +2ms
  cypress:cli { verified: true } +2ms
  cypress:cli is Verified ? true +1ms
  cypress:cli opening from options {"project":"/home/aryzing/workspace/stacks-challenge"} +0ms
  cypress:cli command line arguments ["--project","/home/aryzing/workspace/stacks-challenge"] +0ms
  cypress:cli DISPLAY environment variable is set to :0 on Linux
  cypress:cli Assuming this DISPLAY points at working X11 server,
  cypress:cli Cypress will not spawn own Xvfb
  cypress:cli 
  cypress:cli NOTE: if the X11 server is NOT working, Cypress will exit without explanation,
  cypress:cli   see https://github.com/cypress-io/cypress/issues/4034
  cypress:cli Solution: Unset the DISPLAY variable and try again:
  cypress:cli   DISPLAY= npx cypress run ... +0ms
  cypress:cli needs to start own Xvfb? false +0ms
  cypress:cli spawning, should retry on display problem? true +0ms
  cypress:cli passing DISPLAY :0 +4ms
  cypress:cli spawn args [ '--no-sandbox', '--', '--project', '/home/aryzing/workspace/stacks-challenge', '--cwd', '/home/aryzing/workspace/stacks-challenge', '--userNodePath', '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', '--userNodeVersion', '18.7.0' ] { detached: false, stdio: [ 'inherit', 'inherit', 'pipe' ] } +0ms
  cypress:cli spawning Cypress with executable: /home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress +1ms
  cypress:cli piping child STDERR to process STDERR +2ms
  cypress:ts registering ts-node on directory undefined +0ms
  cypress:ts Running without ts-node hook in environment "production" +33ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/cache +0ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/browsers +31ms
  cypress:server:cypress starting cypress with argv [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress', '--no-sandbox', '--', '--project', '/home/aryzing/workspace/stacks-challenge', '--cwd', '/home/aryzing/workspace/stacks-challenge', '--userNodePath', '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', '--userNodeVersion', '18.7.0' ] +0ms
  cypress:server:args argv array: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress', '--no-sandbox', '--project', '/home/aryzing/workspace/stacks-challenge', '--cwd', '/home/aryzing/workspace/stacks-challenge', '--userNodePath', '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', '--userNodeVersion', '18.7.0' ] +0ms
  cypress:server:args parsed argv options { options: { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0' } } +1ms
  cypress:server:args argv parsed: { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0', invokedFromCli: true } +2ms
  cypress:server:util:proxy found proxy environment variables {} +0ms
  cypress:server:args options { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0', invokedFromCli: true, config: {} } +0ms
  cypress:server:args argv options: { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0', invokedFromCli: true, config: {}, projectRoot: '/home/aryzing/workspace/stacks-challenge' } +0ms
  cypress:server:cypress from argv [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress', '--no-sandbox', '--project', '/home/aryzing/workspace/stacks-challenge', '--cwd', '/home/aryzing/workspace/stacks-challenge', '--userNodePath', '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', '--userNodeVersion', '18.7.0' ] got options { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0', invokedFromCli: true, config: {}, projectRoot: '/home/aryzing/workspace/stacks-challenge' } +5ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production +90ms
[42774:0730/155903.761829:WARNING:bluez_dbus_manager.cc(248)] Floss manager not present, cannot set Floss enable/disable.
  cypress:server:cypress starting in mode interactive with options { _: [ '/home/aryzing/.cache/Cypress/10.3.1/Cypress/Cypress' ], sandbox: false, project: '/home/aryzing/workspace/stacks-challenge', cwd: '/home/aryzing/workspace/stacks-challenge', userNodePath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node', userNodeVersion: '18.7.0', invokedFromCli: true, config: {}, projectRoot: '/home/aryzing/workspace/stacks-challenge' } +103ms
  cypress:server:cypress running Electron currently +0ms
[42932:0730/155903.794937:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.cloudLatestRunUpdateSpecData +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.cloudProjectBySlug +1ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.cloudProjectsBySlugs +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.cloudSpecByPath +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.cloudViewer +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for Query.versions +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for CurrentProject.cloudProject +1ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for RemoteFetchableCloudProjectResult.data +1ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Adding nexusDeferIfNotLoadedPlugin for RemoteFetchableCloudProjectSpecResult.data +1ms
  cypress:server:browsers:utils getBrowsers +0ms
  cypress:launcher:detect detecting if the following browsers are present [ { name: 'chrome', family: 'chromium', channel: 'stable', displayName: 'Chrome', versionRegex: /Google Chrome (\S+)/m, binary: [ 'google-chrome', 'chrome', 'google-chrome-stable' ], minSupportedVersion: 64 }, { name: 'chromium', family: 'chromium', channel: 'stable', displayName: 'Chromium', versionRegex: /Chromium (\S+)/m, binary: [ 'chromium-browser', 'chromium' ], minSupportedVersion: 64 }, { name: 'chrome', family: 'chromium', channel: 'beta', displayName: 'Chrome Beta', versionRegex: /Google Chrome (\S+) beta/m, binary: 'google-chrome-beta', minSupportedVersion: 64 }, { name: 'chrome', family: 'chromium', channel: 'canary', displayName: 'Canary', versionRegex: /Google Chrome Canary (\S+)/m, binary: 'google-chrome-canary', minSupportedVersion: 64 }, { name: 'firefox', family: 'firefox', channel: 'stable', displayName: 'Firefox', versionRegex: /^Mozilla Firefox ([^\sab]+)$/m, binary: 'firefox', minSupportedVersion: 86 }, { name: 'firefox', family: 'firefox', channel: 'dev', displayName: 'Firefox Developer Edition', versionRegex: /^Mozilla Firefox (\S+b\S*)$/m, binary: [ 'firefox-developer-edition', 'firefox' ], minSupportedVersion: 86 }, { name: 'firefox', family: 'firefox', channel: 'nightly', displayName: 'Firefox Nightly', versionRegex: /^Mozilla Firefox (\S+a\S*)$/m, binary: [ 'firefox-nightly', 'firefox-trunk' ], minSupportedVersion: 86 }, { name: 'edge', family: 'chromium', channel: 'stable', displayName: 'Edge', versionRegex: /Microsoft Edge (\S+)/m, binary: [ 'edge', 'microsoft-edge' ], minSupportedVersion: 79 }, { name: 'edge', family: 'chromium', channel: 'canary', displayName: 'Edge Canary', versionRegex: /Microsoft Edge Canary (\S+)/m, binary: 'edge-canary', minSupportedVersion: 79 }, { name: 'edge', family: 'chromium', channel: 'beta', displayName: 'Edge Beta', versionRegex: /Microsoft Edge Beta (\S+)/m, binary: 'edge-beta', minSupportedVersion: 79 }, { name: 'edge', family: 'chromium', channel: 'dev', displayName: 'Edge Dev', versionRegex: /Microsoft Edge Dev (\S+)/m, binary: [ 'edge-dev', 'microsoft-edge-dev' ], minSupportedVersion: 79 } ] +0ms
  cypress:data-context:sources:GitDataSource config: { isRunMode: false, projectRoot: '/home/aryzing/workspace/stacks-challenge', onError: [Function (anonymous)], onBranchChange: [Function: onBranchChange], onGitInfoChange: [Function: onGitInfoChange] } +0ms
  cypress:scaffold-config:detect Detected cypress.config.ts - using TS +0ms
  cypress:server:util:editors get user editor +0ms
  cypress:server:saved_state making saved state from /home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/packages/server +0ms
  cypress:server:saved_state missing project path, looking for project here +0ms
  cypress:lifecycle:ProjectConfigIpc fork child process { CHILD_PROCESS_FILE_PATH: '/home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/node_modules/@packages/server/lib/plugins/child/require_async_child.js', configProcessArgs: [ '--projectRoot', '/home/aryzing/workspace/stacks-challenge', '--file', '/home/aryzing/workspace/stacks-challenge/cypress.config.ts' ], childOptions: { stdio: 'pipe', cwd: '/home/aryzing/workspace/stacks-challenge', execPath: '/home/aryzing/.nvm/versions/node/v18.7.0/bin/node' } } +0ms
  cypress:lifecycle:ProjectConfigIpc trigger the load of the file +13ms
  cypress:server:saved_state making saved state from /home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/packages/server +30ms
  cypress:server:saved_state for project path /home/aryzing/workspace/stacks-challenge +0ms
  cypress:server:saved_state state path for project /home/aryzing/workspace/stacks-challenge +26ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/projects/stacks-challenge-0266e2696a1adfdddd70cc3c874c2a27/state.json +463ms
  cypress:server:saved_state full state path /home/aryzing/.config/Cypress/cy/production/projects/stacks-challenge-0266e2696a1adfdddd70cc3c874c2a27/state.json +1ms
  cypress:server:saved_state making new state file around /home/aryzing/.config/Cypress/cy/production/projects/stacks-challenge-0266e2696a1adfdddd70cc3c874c2a27/state.json +0ms
(node:43278) ExperimentalWarning: Custom ESM Loaders is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
(node:43278) ExperimentalWarning: The Node.js specifier resolution flag is experimental. It could change or be removed at any time.
  cypress:server:saved_state state path for global mode +28ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +28ms
  cypress:server:saved_state full state path /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +0ms
  cypress:server:saved_state making new state file around /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +0ms
  cypress:server:util:process_profiler current & mean memory and CPU usage by process group:
  cypress:server:util:process_profiler ┌─────────┬───────────────────┬──────────────┬────────────────┬────────────┬────────────────┬──────────┬──────────────┬─────────────┐
  cypress:server:util:process_profiler │ (index) │       group       │ processCount │      pids      │ cpuPercent │ meanCpuPercent │ memRssMb │ meanMemRssMb │ maxMemRssMb │
  cypress:server:util:process_profiler ├─────────┼───────────────────┼──────────────┼────────────────┼────────────┼────────────────┼──────────┼──────────────┼─────────────┤
  cypress:server:util:process_profiler │    0    │     'cypress'     │      1       │    '42774'     │     0      │       0        │  150.1   │    150.1     │    150.1    │
  cypress:server:util:process_profiler │    1    │ 'electron-shared' │      2       │ '42776, 42777' │     0      │       0        │  94.29   │    94.29     │    94.29    │
  cypress:server:util:process_profiler │    2    │      'other'      │      2       │ '42917, 42918' │     0      │       0        │   2.51   │     2.51     │    2.51     │
  cypress:server:util:process_profiler │    3    │      'TOTAL'      │      5       │      '-'       │     0      │       0        │  246.9   │    246.9     │    246.9    │
  cypress:server:util:process_profiler └─────────┴───────────────────┴──────────────┴────────────────┴────────────┴────────────────┴──────────┴──────────────┴─────────────┘ +0ms
  cypress:data-context:sources:GitDataSource On current branch feat-add-cypress +121ms
  cypress:server:windows creating electron window with options { backgroundColor: 'white', width: 1565, height: 1001, minWidth: 597, minHeight: 400, x: 680, y: 450, type: 'INDEX', devTools: false, trackState: { width: 'appWidth', height: 'appHeight', x: 'appX', y: 'appY', devTools: 'isAppDevToolsOpen' }, onBlur: [Function: onBlur], onFocus: [Function: onFocus], onClose: [Function: onClose], show: true, frame: true, transparent: false, icon: NativeImage { toPNG: [Function: toPNG], toJPEG: [Function: toJPEG], toBitmap: [Function: toBitmap], getBitmap: [Function: getBitmap], getScaleFactors: [Function: getScaleFactors], getNativeHandle: [Function: getNativeHandle], toDataURL: [Function: toDataURL], isEmpty: [Function: isEmpty], getSize: [Function: getSize], setTemplateImage: [Function: setTemplateImage], isTemplateImage: [Function: isTemplateImage], isMacTemplateImage: [Getter/Setter], resize: [Function: resize], crop: [Function: crop], getAspectRatio: [Function: getAspectRatio], addRepresentation: [Function: addRepresentation] }, webPreferences: { contextIsolation: true, partition: null, webSecurity: false, nodeIntegration: false, backgroundThrottling: false }, url: 'http://localhost:38167/__launchpad/index.html', contextMenu: false, recordFrameRate: null, onCrashed: [Function: onCrashed], onNewWindow: [Function: onNewWindow] } +0ms
Error [ERR_LOADER_CHAIN_INCOMPLETE]: "file:///home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/node_modules/ts-node/esm/transpile-only.mjs 'resolve'" did not call the next hook in its chain and did not explicitly signal a short circuit. If this is intentional, include `shortCircuit: true` in the hook's return.
    at new NodeError (node:internal/errors:387:5)
    at ESMLoader.resolve (node:internal/modules/esm/loader:860:13)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:7)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:541:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ERR_LOADER_CHAIN_INCOMPLETE'
}
  cypress:server:util:editors user has the following editors: [ { id: 'code', binary: 'code', name: 'Visual Studio Code' } ] +293ms
  cypress:server:saved_state making saved state from /home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/packages/server +208ms
  cypress:server:saved_state missing project path, looking for project here +0ms
  cypress:server:saved_state state path for global mode +2ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +210ms
  cypress:server:saved_state full state path /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +0ms
  cypress:server:util:editors saved preferred editor: undefined +3ms
  cypress:server:util:editors return available editors: [ { id: 'computer', name: 'File System', binary: 'computer' }, { id: 'code', name: 'Visual Studio Code', binary: 'code' } ] +0ms
  cypress:server:saved_state making saved state from /home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/packages/server +2ms
  cypress:server:saved_state missing project path, looking for project here +0ms
  cypress:server:saved_state state path for global mode +4ms
  cypress:server:appdata path: /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +6ms
  cypress:server:saved_state full state path /home/aryzing/.config/Cypress/cy/production/projects/__global__/state.json +1ms
  cypress:launcher:linux firefox is running as a snap, changing profile path +0ms
  cypress:launcher:linux firefox is running as a snap, changing profile path +132ms
  cypress:launcher:linux Could not extract version from stdout using regex: { stdout: 'Mozilla Firefox 103.0', versionRegex: /^Mozilla Firefox (\S+b\S*)$/m } +0ms
  cypress:server:browsers:utils found browsers { browsers: [ { name: 'firefox', family: 'firefox', channel: 'stable', displayName: 'Firefox', version: '103.0', path: 'firefox', profilePath: '/home/aryzing/snap/firefox/current', minSupportedVersion: 86, majorVersion: 103 } ] } +517ms
  cypress:server:browsers:utils adding Electron browser { name: 'electron', channel: 'stable', family: 'chromium', displayName: 'Electron', version: '100.0.4896.160', path: '', majorVersion: 100, info: 'Electron is the default browser that comes with Cypress. This is the default browser that runs in headless mode. Selecting this browser is useful when debugging. The version number indicates the underlying Chromium version that Electron uses.' } +1ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Racing execution for Query.versions +587ms
  cypress:network:agent addRequest called { isHttps: true, href: 'https://registry.npmjs.org/cypress' } +0ms
  cypress:network:connect beginning getAddress { hostname: 'registry.npmjs.org', port: 80 } +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Racing execution for Query.cloudViewer +20ms
  cypress:graphql:remoteSchemaWrapped executing: {"operationName":"HeaderBar_HeaderBarQuery_cloudViewer","requestPolicy":"cache-first"} +0ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Racing Query.cloudViewer resolved immediately +5ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Query.versions did not resolve immediately +1ms
  cypress:network:agent addRequest called { isHttps: true, href: 'https://download.cypress.io/desktop.json' } +10ms
  cypress:network:connect beginning getAddress { hostname: 'download.cypress.io', port: 80 } +9ms
  cypress:network:connect got addresses { hostname: 'registry.npmjs.org', port: 80, addresses: [ { address: '104.16.20.35', family: 4 }, { address: '104.16.21.35', family: 4 }, { address: '104.16.19.35', family: 4 }, { address: '104.16.26.35', family: 4 }, { address: '104.16.25.35', family: 4 }, { address: '104.16.23.35', family: 4 }, { address: '104.16.24.35', family: 4 }, { address: '104.16.22.35', family: 4 }, { address: '104.16.16.35', family: 4 }, { address: '104.16.18.35', family: 4 }, { address: '104.16.17.35', family: 4 }, { address: '104.16.27.35', family: 4 }, { address: '2606:4700::6810:1523', family: 6 }, { address: '2606:4700::6810:1323', family: 6 }, { address: '2606:4700::6810:1823', family: 6 }, { address: '2606:4700::6810:1a23', family: 6 }, { address: '2606:4700::6810:1223', family: 6 }, { address: '2606:4700::6810:1123', family: 6 }, { address: '2606:4700::6810:1623', family: 6 }, { address: '2606:4700::6810:1b23', family: 6 }, { address: '2606:4700::6810:1923', family: 6 }, { address: '2606:4700::6810:1723', family: 6 }, { address: '2606:4700::6810:1023', family: 6 }, { address: '2606:4700::6810:1423', family: 6 } ] } +4ms
  cypress:network:connect got addresses { hostname: 'download.cypress.io', port: 80, addresses: [ { address: '172.67.25.250', family: 4 }, { address: '104.22.10.239', family: 4 }, { address: '104.22.11.239', family: 4 }, { address: '2606:4700:10::6816:aef', family: 6 }, { address: '2606:4700:10::ac43:19fa', family: 6 }, { address: '2606:4700:10::6816:bef', family: 6 } ] } +15ms
  cypress:network:agent got family { family: 4, href: 'https://registry.npmjs.org/cypress' } +23ms
  cypress:network:agent got family { family: 4, href: 'https://download.cypress.io/desktop.json' } +13ms
  cypress:data-context:sources:VersionsDataSource retrieving latest version information with headers: { 'Content-Type': 'application/json', 'x-cypress-version': '10.3.1', 'x-os-name': 'linux', 'x-arch': 'x64', 'x-initial-launch': 'true', 'x-machine-id': 'f7fa3d41187792cb7c538ee98f58a20c5d9040975ef10fd9c79ee9ddd0795f16' } +0ms
  cypress:data-context:sources:VersionsDataSource latest version information: { name: 'Cypress', version: '10.3.1', packages: { mac: { url: 'https://cdn.cypress.io/desktop/10.3.1/darwin-x64/cypress.zip' }, linux64: { url: 'https://cdn.cypress.io/desktop/10.3.1/linux-x64/cypress.zip' }, darwin: { url: 'https://cdn.cypress.io/desktop/10.3.1/darwin-x64/cypress.zip' }, linux: { url: 'https://cdn.cypress.io/desktop/10.3.1/linux-x64/cypress.zip' }, 'darwin-x64': { url: 'https://cdn.cypress.io/desktop/10.3.1/darwin-x64/cypress.zip' }, 'darwin-arm64': { url: 'https://cdn.cypress.io/desktop/10.3.1/darwin-arm64/cypress.zip' }, 'linux-x64': { url: 'https://cdn.cypress.io/desktop/10.3.1/linux-x64/cypress.zip' }, 'linux-arm64': { url: 'https://cdn.cypress.io/desktop/10.3.1/linux-arm64/cypress.zip' }, 'win32-x64': { url: 'https://cdn.cypress.io/desktop/10.3.1/win32-x64/cypress.zip' } } } +3ms
  cypress:data-context:sources:VersionsDataSource NPM release dates received { modified: '2022-07-19T16:51:56.330Z' } +582ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Racing Query.versions eventually resolved with { latest: { id: '10.3.1', version: '10.3.1', released: '2022-07-18T19:24:24.458Z' }, current: { id: '10.3.1', version: '10.3.1', released: '2022-07-18T19:24:24.458Z' } } HeaderBar_HeaderBarQuery +897ms
  cypress:graphql:nexusDeferIfNotLoadedPlugin Resolving versions for pushFragment with {"latest":{"id":"10.3.1","version":"10.3.1","released":"2022-07-18T19:24:24.458Z"},"current":{"id":"10.3.1","version":"10.3.1","released":"2022-07-18T19:24:24.458Z"}} +1ms
  cypress:data-context:sources:GraphQLDataSource pushFragment value {"data":{"versions":{"current":{"id":"10.3.1","version":"10.3.1","released":"2022-07-18T19:24:24.458Z","__typename":"Version"},"latest":{"id":"10.3.1","version":"10.3.1","released":"2022-07-18T19:24:24.458Z","__typename":"Version"},"__typename":"VersionData"}}} +0ms
  cypress:server:util:process_profiler current & mean memory and CPU usage by process group:
  cypress:server:util:process_profiler ┌─────────┬───────────────────┬──────────────┬──────────────────────────────┬────────────┬────────────────┬──────────┬──────────────┬─────────────┐
  cypress:server:util:process_profiler │ (index) │       group       │ processCount │             pids             │ cpuPercent │ meanCpuPercent │ memRssMb │ meanMemRssMb │ maxMemRssMb │
  cypress:server:util:process_profiler ├─────────┼───────────────────┼──────────────┼──────────────────────────────┼────────────┼────────────────┼──────────┼──────────────┼─────────────┤
  cypress:server:util:process_profiler │    0    │ 'electron-shared' │      4       │ '42776, 42932, 42777, 43543' │    0.01    │       0        │  285.87  │    190.08    │   285.87    │
  cypress:server:util:process_profiler │    1    │     'cypress'     │      1       │           '42774'            │    0.78    │      0.39      │  198.92  │    174.51    │   198.92    │
  cypress:server:util:process_profiler │    2    │    'launchpad'    │      1       │           '43592'            │     0      │       0        │  110.88  │    110.88    │   110.88    │
  cypress:server:util:process_profiler │    3    │      'other'      │      2       │        '43869, 43870'        │     0      │       0        │   2.55   │     2.53     │    2.55     │
  cypress:server:util:process_profiler │    4    │      'TOTAL'      │      8       │             '-'              │    0.79    │      0.39      │  598.21  │    422.56    │   598.21    │
  cypress:server:util:process_profiler └─────────┴───────────────────┴──────────────┴──────────────────────────────┴────────────┴────────────────┴──────────┴──────────────┴─────────────┘ +10s

```

</details>

To my untrained eye, this error in the logs caught my attention, although I'm not sure if that's what's causing the infinite loading screen,

```
Error [ERR_LOADER_CHAIN_INCOMPLETE]: "file:///home/aryzing/.cache/Cypress/10.3.1/Cypress/resources/app/node_modules/ts-node/esm/transpile-only.mjs 'resolve'" did not call the next hook in its chain and did not explicitly signal a short circuit. If this is intentional, include `shortCircuit: true` in the hook's return.
    at new NodeError (node:internal/errors:387:5)
    at ESMLoader.resolve (node:internal/modules/esm/loader:860:13)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:7)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:541:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ERR_LOADER_CHAIN_INCOMPLETE'
}
```
